code/modules/mob/living/simple_animal/abnormality/waw/thunder_bird.dm

```diff
@@ -160,12 +161,11 @@ GLOBAL_LIST_EMPTY(zombies)
 			new /obj/effect/temp_visual/kinetic_blast(LT)
 			if(ishuman(L))
 				var/mob/living/carbon/human/H = L
-				H.apply_damage(25,BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
+				H.apply_damage(100,BLACK_DAMAGE, null, H.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
 				H.electrocute_act(1, src, flags = SHOCK_NOSTUN)
-				H.Knockdown(30)
+				//H.Knockdown(30)  no stuns
 			else
-				//simple mobs cannot be stunned; they take more damage
-				L.adjustBlackLoss(150)
+				L.adjustBlackLoss(100)
```

Comment:
You can just use apply damage from above instead of adding if/else.

Reply:
I wasn't really sure about it, but can't I just nest the apply_damage line above the ishuman check since the stuns are irrelevant now?

Reply:
Exactly. The only thing that works on humans only is electrocute act now.